Parsers/ASimRegistryEvent/test/Windows_Event_AsimRegistryEvent_DataTest.csv

```diff
@@ -0,0 +1,11 @@
+﻿Result
+"(0) Error: 1 invalid value(s) (up to 10 listed) in 10 records (100.0%) for field [EventVendor] of type [Enumerated]: [""Microsoft""] (Schema:RegistryEvent)"
+"(0) Error: 1 invalid value(s) (up to 10 listed) in 3 records (30.0%) for field [EventType] of type [Enumerated]: [""RegistryKeyNotify""] (Schema:RegistryEvent)"
```

Comment:
please check this error as well

Reply:
done

Reply:
what was the issue before?

Reply:
Adeed microsoft and trend microsoft in ARM